keyboards/quarkeys/z60/readme.md

```diff
@@ -0,0 +1,22 @@
+# Quarkeys Z60 Keyboard
+
+![Z60](https://static.wixstatic.com/media/72b918_45f82258186646d4bfa57c9d73ccc9bc~mv2.jpeg)
```

Comment:
Please rehost on Imgur.

Reply:
Hi drasma, I tried with Imgur but it has restriction for my region's cell number, so I guess I just delete the Picture in readme.md?

Reply:
Hi, probably stupid to ask, do I need to click the 'Resolve conversation' button?